frontendUiApi/src/main/java/com/virtuslab/gitmachete/frontend/ui/api/table/BaseGraphTable.java

```diff
@@ -11,8 +15,25 @@
 
 public abstract class BaseGraphTable extends JBTable {
   @UIEffect
+  @SuppressWarnings("nullness:method.invocation.invalid") // to allow for setAutoResizeMode despite the object isn't initialized yet
   protected BaseGraphTable(TableModel model) {
     super(model);
+
+    // Without set autoresize off column will expand to whole table width (will not fit the content size)
+    setAutoResizeMode(JBTable.AUTO_RESIZE_OFF);
+  }
+
+  /**
+   * This method that overrides {@link JBTable#prepareRenderer} is responsible for set column size that fit the content
```

Comment:
`for setting`; as a rule of the thumb, there's always a gerund and not infinitive after a preposition like `for`, `of`, `on` etc. (only `to` is usually followed by an infinitive)

Reply:
`that fits` (?) since a singular `size` is the subject